openmp/libomptarget/plugins-nextgen/common/include/GlobalHandler.h

```diff
@@ -58,6 +60,22 @@ class GlobalTy {
   void setPtr(void *P) { Ptr = P; }
 };
 
+typedef void *IntPtrT;
```

Comment:
What's the utility of this?

Reply:
`IntPtrT` isn't defined in `profile/InstrProfData.inc`. There are multiple examples of this type being defined. before InstProfData.inc is included. I'm not entirely sure what its for, but I'm assuming its to account for different platforms with different integer sizes.

Reply:
Okay. you should use the C++ `using` keyword instead of C's `typedef.